server/api/users.js

```diff
@@ -131,7 +135,7 @@ router.post('users.s3Upload', auth(), async ctx => {
       asset: {
         contentType,
         name,
-        url,
+        url: `/api/images.info?id=${id}`,
```

Comment:
this is a question I had. Here I'm returning the url in the form required to hit the image proxy endpoint, so that it can be written in the markdown as the proxy link, but it is being saved in the `Attachments` table as the full url. This is working fine as it is here, but just an open question on whether this is the best way to do it? i.e. what url should be saved vs returned to the front-end and is it different depending on what mode we're in with the `AWS_S3_ACL` value.